src/game/base.ts

```diff
@@ -180,8 +180,11 @@ export class Game implements IGame {
 
     public nextRoundNumber() {
         if (this.difficulty === GameDifficulty.Noob) {
-            console.log("base - incrementRoundNumber - noob");
-            return 1;
+            console.log(
+                "base - decrementRoundNumber#noob - ",
+                this.state.currentCount,
+            );
+            return this.state.currentCount - 1;
```

Comment:
Why decrement?

Reply:
- to never fulfill the end of game check - endless mode require this
- to have 'knowledge' about round number used for computing average round time